lib/project_types/node/commands/create.rb

```diff
@@ -112,6 +113,11 @@ def build(name)
           @ctx.debug(e)
         end
       end
+
+      def partners_endpoint
+        return 'https://partners.shopify.com' if ENV[LOCAL_DEBUG].nil?
```

Comment:
This is fairly minor, but I think it would be nice to flip these checks, e.g.

```
return 'https://partners.myshopify.io' if ENV[LOCAL_DEBUG]
'https://partners.shopify.com'
```

Often the early returns in a method are handle special cases, sometimes known as a [Guard clause](https://wiki.c2.com/?GuardClause).